motionSetPointsApp/Db/motionSetPoints10Axis.substitutions

```diff
@@ -0,0 +1,93 @@
+	{ "\$(P)", "8", "\$(AXIS9)", "\$(LOOKUP)" }
+	{ "\$(P)", "9", "\$(AXIS10)", "\$(LOOKUP)" }
+}
+
+file $(UTILITIES)/db/unit_setter.template { 
+  pattern 
+    {P,    FROM, TO}
+
+    {"", "\$(AXIS1).EGU", "\$(P)COORD0:NO_OFF"}
+    {"", "\$(AXIS1).EGU", "\$(P)COORD0:OFFSET"}
+	{"", "\$(AXIS1).EGU", "\$(P)COORD0"}
+	{"", "\$(AXIS1).EGU", "\$(P)COORD0:RBV"}
+	{"", "\$(AXIS1).EGU", "\$(P)COORD0:RBV:OFF"}
+	{"", "\$(AXIS1).EGU", "\$(P)COORD0:SET:RBV"}
+	
+	{"", "\$(AXIS2).EGU", "\$(P)COORD1:NO_OFF"}
+    {"", "\$(AXIS2).EGU", "\$(P)COORD1:OFFSET"}
+	{"", "\$(AXIS2).EGU", "\$(P)COORD1"}
+	{"", "\$(AXIS2).EGU", "\$(P)COORD1:RBV"}
+	{"", "\$(AXIS2).EGU", "\$(P)COORD1:RBV:OFF"}
+	{"", "\$(AXIS2).EGU", "\$(P)COORD1:SET:RBV"}
+	
+    {"", "\$(AXIS3).EGU", "\$(P)COORD2:NO_OFF"}
+    {"", "\$(AXIS3).EGU", "\$(P)COORD2:OFFSET"}
+	{"", "\$(AXIS3).EGU", "\$(P)COORD2"}
+	{"", "\$(AXIS3).EGU", "\$(P)COORD2:RBV"}
+	{"", "\$(AXIS3).EGU", "\$(P)COORD2:RBV:OFF"}
+	{"", "\$(AXIS3).EGU", "\$(P)COORD2:SET:RBV"}
+	
+	{"", "\$(AXIS4).EGU", "\$(P)COORD3:NO_OFF"}
+    {"", "\$(AXIS4).EGU", "\$(P)COORD3:OFFSET"}
+	{"", "\$(AXIS4).EGU", "\$(P)COORD3"}
+	{"", "\$(AXIS4).EGU", "\$(P)COORD3:RBV"}
+	{"", "\$(AXIS4).EGU", "\$(P)COORD3:RBV:OFF"}
+	{"", "\$(AXIS4).EGU", "\$(P)COORD3:SET:RBV"}
+	
+	{"", "\$(AXIS5).EGU", "\$(P)COORD4:NO_OFF"}
+    {"", "\$(AXIS5).EGU", "\$(P)COORD4:OFFSET"}
+	{"", "\$(AXIS5).EGU", "\$(P)COORD4"}
+	{"", "\$(AXIS5).EGU", "\$(P)COORD4:RBV"}
+	{"", "\$(AXIS5).EGU", "\$(P)COORD4:RBV:OFF"}
+	{"", "\$(AXIS5).EGU", "\$(P)COORD4:SET:RBV"}
+	               
+	{"", "\$(AXIS6).EGU", "\$(P)COORD5:NO_OFF"}
+    {"", "\$(AXIS6).EGU", "\$(P)COORD5:OFFSET"}
+	{"", "\$(AXIS6).EGU", "\$(P)COORD5"}
+	{"", "\$(AXIS6).EGU", "\$(P)COORD5:RBV"}
+	{"", "\$(AXIS6).EGU", "\$(P)COORD5:RBV:OFF"}
+	{"", "\$(AXIS6).EGU", "\$(P)COORD5:SET:RBV"}
+	
+    {"", "\$(AXIS7).EGU", "\$(P)COORD6:NO_OFF"}
+    {"", "\$(AXIS7).EGU", "\$(P)COORD6:OFFSET"}
+	{"", "\$(AXIS7).EGU", "\$(P)COORD6"}
+	{"", "\$(AXIS7).EGU", "\$(P)COORD6:RBV"}
+	{"", "\$(AXIS7).EGU", "\$(P)COORD6:RBV:OFF"}
+	{"", "\$(AXIS7).EGU", "\$(P)COORD6:SET:RBV"}
+	               
+	{"", "\$(AXIS8).EGU", "\$(P)COORD7:NO_OFF"}
+    {"", "\$(AXIS8).EGU", "\$(P)COORD7:OFFSET"}
+	{"", "\$(AXIS8).EGU", "\$(P)COORD7"}
+	{"", "\$(AXIS8).EGU", "\$(P)COORD7:RBV"}
+	{"", "\$(AXIS8).EGU", "\$(P)COORD7:RBV:OFF"}
+	{"", "\$(AXIS8).EGU", "\$(P)COORD7:SET:RBV"}
+	
+	{"", "\$(AXIS9).EGU", "\$(P)COORD8:NO_OFF"}
+    {"", "\$(AXIS9).EGU", "\$(P)COORD8:OFFSET"}
+	{"", "\$(AXIS9).EGU", "\$(P)COORD8"}
+	{"", "\$(AXIS9).EGU", "\$(P)COORD8:RBV"}
+	{"", "\$(AXIS9).EGU", "\$(P)COORD8:RBV:OFF"}
+	{"", "\$(AXIS9).EGU", "\$(P)COORD8:SET:RBV"}
+	
+	{"", "\$(AXIS10).EGU", "\$(P)COORD9:NO_OFF"}
+    {"", "\$(AXIS10).EGU", "\$(P)COORD9:OFFSET"}
+	{"", "\$(AXIS10).EGU", "\$(P)COORD9"}
+	{"", "\$(AXIS10).EGU", "\$(P)COORD9:RBV"}
+	{"", "\$(AXIS10).EGU", "\$(P)COORD9:RBV:OFF"}
+	{"", "\$(AXIS10).EGU", "\$(P)COORD9:SET:RBV"}
```

Comment:
There is dbLoadRecordsLoop that I wrote https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Tips-and-Tricks and there is also lua

Reply:
Yh, I looked at dbLoadRecordsLoop but I don't think it works because we need to parameterise the macro names. I also thought lua was complicating an already complicated ticket. I will have a think though

Reply:
Yeah it would be nice to be able to do this, but I don't think we should overcomplicate and stretch the ticket out if it will do that

Reply:
A month later and still can't think of a good way to do this that's going to make it much more readable without adding more complexity.